.buildkite/pipelines/pull_request/defend_workflows.yml

```diff
@@ -19,7 +19,7 @@ steps:
       queue: n2-16-virt
```

Comment:
Do these all need 16 cores?  Is 4 cores / 16 gb of memory sufficient via n2-4-virt?

Reply:
checking :)